CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
    * FIXED: Fix heading on small edge [#3114](https://github.com/valhalla/valhalla/pull/3114)
    * FIXED: Added support for `access=psv`, which disables routing on these nodes and edges unless the mode is taxi or bus [#3107](https://github.com/valhalla/valhalla/pull/3107)
    * FIXED: Fixed U-turns through service roads [#3082](https://github.com/valhalla/valhalla/pull/3082)
+   * FIXED: Added forgotten penalties for kLivingStreet and kTrack for pedestrian costing model [#3116](https://github.com/valhalla/valhalla/pull/3116)
```

Comment:
one thing this pr didnt do is check what happens in RAD now that these costs are added. for tracks i suppose it made no difference because the value was set to .5 by default but the default of .6 for living streets should have some impact in some areas.

Reply:
i think its ok to merge anyway though as this was clearly an oversight. if the pedestrian cost needs tuning we can do that in a separate pr. anyone else have a different opinion or some time to check RAD?

Reply:
@kevinkreiser .6 for living streets wasn't changed in this PR. It's been this way since the day I introduced this feature.

Reply:
right i get that but it wasnt being used before in pedestrian transition costs which means we do have a potential for pedestrian paths to change right?

Reply:
Uhm, sorry, I'm slow, not getting what you are concerned about... .6 means zero penalty, so no change was expected now regarding changes in transition cost

Reply:
no you arent slow, i see it now. the penalty is set to 0 in the ternary in dynamiccost.cc when its >= to .5. for some reason i was looking at the factor rather than the penalty. sorry about that!